integration-tests/fromMnemonic-send-tez.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { InMemorySigner } from "@taquito/signer";
+import { CONFIGS } from "./config";
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+  const Tezos = lib;
+   describe(`Test fromMnemonic instantiation with rpc: ${rpc}`, () => {
+
+    beforeEach(async (done) => {
+      await setup()
+      done()
+    })
+    it('Should create an InMemorySigner with the fromMnemonic method and transfer tez to an account', async (done) => {
```

Comment:
Can we have a test case for 'secp256k1' and 'p256' also?

Reply:
👍